frontend/modelarchive/src/main/java/com/amazonaws/ml/mms/archive/ModelArchive.java

```diff
@@ -217,6 +217,11 @@ private static ModelArchive load(String url, File dir, boolean extracted)
                 }
             }
 
+            if (manifest.getModel().getModelName() == null
```

Comment:
We purposely all empty model name here, to support model archive without MANIFEST.xml. The model name checking can be default in load model management API call.
And for initial model loading, we assign model name and override in the model archive anyway

Reply:
I probably need to understand this requirement for non MANIFEST model archive that was done.  I will discuss this with you offline.

Reply:
This is required as Manifest is created empty and the model-name is not set in the Manifest. We could go in one of the two routes,
1. We could add a constructor to Manifest, which initializes the Model with a model-name, which in this case is URL.
(or)
2. We could go with this change, where we set the modelName explicitly after creating the model archive.